src/views/Account.vue

```diff
@@ -232,11 +232,11 @@ export default {
 
     if (this.balance > 0) {
```

Comment:
Why 2 places for this?

Reply:
- One at when use click on actual asset - Ex: Click on BTC 
- One at as soon as they landed on overview page - Update balance